src/parseTools.js

```diff
@@ -1222,10 +1222,16 @@ function makeAsmImportsAccessInPthread(variable) {
   return variable;
 }
 
+// Returns true if the given function is exported out in the generated JS Module object.
 function hasExportedFunction(func) {
   return Object.keys(EXPORTED_FUNCTIONS).indexOf(func) != -1;
 }
 
+// Returns true if LLVM produced the given function into compiled code as a Wasm export.
+function hasLLVMExportedFunction(func) {
+  return LLVM_EXPORTS.indexOf(func) != -1;
```

Comment:
I think we can do `LLVM_EXPORTS.includes(func)` these days.  (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes)